<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobList</class>
 <widget class="QDialog" name="JobList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job list</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTableView" name="tblJobs">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create job(s) (F1)</string>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Delete job(s) (F2)</string>
       </property>
       <property name="shortcut">
        <string>F2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh (F3)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearAll">
       <property name="text">
        <string>Clear ALL jobs (F4)</string>
       </property>
       <property name="shortcut">
        <string>F4</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <disabledon>:/images/smallIcons_1461.ico</disabledon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../opsipackagebuilder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>JobList</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
